src/derivative_wrappers.jl

```diff
@@ -102,6 +102,18 @@ function DiffEqBase.build_jac_config(alg::OrdinaryDiffEqAlgorithm,f,uf,du1,uprev
   jac_config
 end
 
+get_chunksize(jac_config::ForwardDiff.JacobianConfig{T,V,N,D}) where {T,V,N,D} = N
+
+function DiffEqBase.resize_jac_config!(jac_config::ForwardDiff.JacobianConfig, uf, du1, z, alg, i)
+  prev_chunksize = get_chunksize(jac_config)
+  ForwardDiff.JacobianConfig(uf,du1,z,ForwardDiff.Chunk{prev_chunksize}())
```

Comment:
did you resize these things elsewhere?

Reply:
Oh yes I forgot to tell about something. https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/misc_utils.jl#L59
This line fails when `autodiff=false` as we don't have duals in `DiffEqDiffTools.JacobainCache`. `full_cache` is used in `resize!` so it fails for `Rosenbrock23(;autodiff=false)` on master. Now that I resize them explicitly, should I remove that from `cache` macro?

Reply:
Yes